src/actions/action.ts

```diff
@@ -224,10 +233,35 @@ export class Action extends Base implements IAction, ILocalizableOwner {
     this.locTitleValue.onStringChanged.add(this.locTitleChanged);
     this.locTitleChanged();
   }
+  public get locTitleName(): string {
+    return this.locTitle.localizationName;
+  }
+  public set locTitleName(val: string) {
+    this.locTitle.localizationName = val;
+  }
+  public locStrsChanged(): void {
+    super.locStrsChanged();
+    this.locTooltipChanged();
+    if (!!this.popupModel && !!this.popupModel.contentComponentData && !!this.popupModel.contentComponentData.model) {

```

Comment:
This probably is better to move to ActionDropdownViewModel class owns popupModel

Reply:
Unfortunately it is a wrapper on action and it doesn't have locStrsChanged.
`export class ActionDropdownViewModel {`
...
```js
  constructor(private item: Action) {
    this.setupPopupCallbacks();
  }
```
I don't like the code myself. However, it is better to have it here than in Creator.